fastserde/avro-fastserde-tests-common/src/test/avro/fastSerdeEnums.avsc

```diff
@@ -0,0 +1,40 @@
+{
+  "type": "record",
```

Comment:
Can't we use the preexisting TestRecord with TestEnum?

Reply:
Well, we could but it's missing map of enums and it's widely used so it seems to be better to create enum-specific avro schema.